src/main/resources/attributes.yaml

```diff
@@ -1,187 +1,184 @@
-  - border-bottom-color
-  - border-bottom-left-radius
-  - border-bottom-right-radius
-  - border-bottom-width
-  - border-collapse
-  - border-image-outset
-  - border-image-repeat
-  - border-image-slice
-  - border-image-source
-  - border-image-width
-  - border-left-color
-  - border-left-style
-  - border-left-width
-  - border-radius
-  - border-right-color
-  - border-right-style
-  - border-right-width
-  - border-spacing
-  - border-top-color
-  - border-top-left-radius
-  - border-top-right-radius
-  - border-top-style
-  - border-top-width
-  - bottom
-  - box-decoration-break
-  - box-shadow
-  - box-sizing
-  - break-after
-  - break-before
-  - break-inside
-  - caption-side
-  - caret-color
-  - clear
-  - clip
-  - color
-  - column-count
-  - column-fill
-  - column-gap
-  - column-rule-color
-  - column-rule-style
-  - column-rule-width
-  - column-span
-  - column-width
-  - content
-  - counter-increment
-  - counter-reset
-  - direction
-  - display
-  - empty-cells
-  - filter
-  - flex-basis
-  - flex-direction
-  - flex-grow
-  - flex-shrink
-  - flex-wrap
-  - float
-  - font-family
-  - font-size
-  - line-height
-  - text-align
-  - text-indent
-  - float
-  - font-kerning
-  - font-size
-  - font-size-adjust
-  - font-stretch
-  - font-style
-  - font-variant
-  - font-weight
-  - grid-auto-columns
-  - grid-auto-flow
-  - grid-auto-rows
-  - grid-column-end
-  - grid-column-start
-  - grid-row-end
-  - grid-row-start
-  - grid-template-areas
-  - grid-template-columns
-  - grid-template-rows
-  - grid-row-gap
-  - grid-column-gap
-  - hanging-punctuation
-  - hyphens
-  - isolation
-  - justify-content
-  - left
-  - letter-spacing
-  - line-height
-  - list-style-image
-  - list-style-position
-  - list-style-type
-  - margin-top
-  - margin-right
-  - margin-bottom
-  - margin-left
-  - max-height
-  - max-width
-  - min-height
-  - min-width
-  - mix-blend-mode
-  - object-fit
-  - object-position
-  - opacity
-  - order
-  - outline-color
-  - outline-offset
-  - outline-style
-  - outline-width
-  - overflow-x
-  - overflow-y
-  - offsetHeight
-  - offsetWidth
-  - padding-top
-  - padding-right
-  - padding-bottom
-  - padding-left
-  - page-break-after
-  - page-break-before
-  - page-break-inside
-  - perspective
-  #  - perspective-origin
-  - pointer-events
-  - position
-  - quotes
-  - resize
-  - right
-  - scroll-behavior
-  - tab-size
-  - table-layout
-  - text-align
-  - text-align-last
-  - text-decoration-color
-  - text-decoration-line
-  - text-decoration-style
-  - text-indent
-  - text-justify
-  - text-overflow
-  - text-shadow
-  - text-transform
-  - top
-  - transform
-  #  - transform-origin
-  - transform-style
-  - transition-delay
-  - transition-duration
-  - transition-property
-  - transition-timing-function
-  - unicode-bidi
-  - user-select
-  - vertical-align
-  - visibility
-  - white-space
-  - word-break
-  - word-spacing
-  - word-wrap
-  - z-index
+  ? align-content
```

Comment:
Is this `?` part of https://github.com/retest/recheck-web/pull/201/files#diff-2f56b6bae935438614bd6475920f25d8R54

Reply:
As I said, that is the syntax for YAML sets.